inductiva/molecules/scenarios/protein_solvation/protein_solvation.py

```diff
@@ -68,13 +74,24 @@ def simulate(
 
             nsteps_minim: Number of steps for energy minimization.
         """
+
+        #Compute charge if it is not provided
+        if self.charged is None:
+            self.charged = self.compute_charge()
+
+        #Edit commands.json according to the charge of the protein
+        commands = self.read_commands_from_file(
+            os.path.join(self.template_dir, "commands.json"))
+
+        if self.charged:
+            commands[3]["cmd"] = commands[3]["cmd"] + " -maxwarn 1"
```

Comment:
I was thinking this would be done via jinja... to systematize with other approaches, wdyt @fabiocruz?

Reply:
Agreed. You can write if statements in jinja templates, see e.g. [this](https://jinja.palletsprojects.com/en/3.1.x/templates/#if).

Reply:
>
Thanks @david451lima. I'm not sure I get the use of the jinja templating engine here, since your options seem to be constant... Did I miss something?
>
This was done with jinja templating but I changed because @fabiocruz requested it. Did I misunderstand what you meant Fábio?

Reply:
I think there was a misunderstanding, yes. You were replacing variables in the jinja template by constant strings, i.e. strings that did not change with e.g. `self.charged`. This suggested that you didn't need a template after all, a simple json file with commands would suffice.

Now, you are fetching commands from a json file, and then modifying them by hand.

A better solution is to use a jinja template where you pass `self.charged` and use that in an if clause inside the template. Does it make sense?